### DISCUSSION TITLE
Alert email forever

### DISCUSSION THREAD

**Discussion by hanhyeonkyu** [community/community#22003] [New to GitHub]
<p>hello. i am developer.</p>
<p>---------------------------STATUS------------------------------<br>
i received email from github.<br>
it was dependency alert of repository.</p>
<p>but this repository is not important, so i removed this repository.</p>
<p>---------------------------PROBLEM-----------------------------<br>
i already removed repository.<br>
but i received alert email continuously from github.<br>
i don’t want to receive email about this repository.<br>
i feel scared. it is like ghost…;</p>
<p>Can you help me?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>if that is not a spam, you can contact GitHub group for that,</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by hanhyeonkyu** [comment]
<p><a class="mention" href="/jdevstatic">@jdevstatic</a> Thank you ^^ i send this issue to support team</p>
